test/onnx/pytorch_test_common.py

```diff
@@ -157,6 +158,24 @@ def wrapper(self, *args, **kwargs):
     return skip_dec
 
 
+def skip_unsupported_min_ort_version(min_opset_version: str):
+    def skip_dec(func):
+        @functools.wraps(func)
+        def wrapper(self, *args, **kwargs):
+            if (
+                packaging.version.parse(self.ort_version).release
+                < packaging.version.parse(min_opset_version).release
+            ):
+                raise unittest.SkipTest(
+                    f"Unsupported onnxruntime version: {self.ort_version} < {min_opset_version}"
```

Comment:
I would make it clear on what is not supported. This sentence can be read both ways (too new or too old) Maybe something like "ONNX Runtime version ... is older than required version ..." ?